cmd/add.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/railwayapp/cli/entity"
+	"github.com/railwayapp/cli/ui"
+)
+
+func stringInSlice(a string, list []*string) bool {
+    for _, b := range list {
+        if *b == a {
```

Comment:
I know you're likely to replace with an array of strings, but in the future, when you deref something, really should have a nil check. You're asking for panics here